### ISSUE TITLE
Character Armory

### ISSUE THREAD

**Issue by Levitanious** [Levitanious/tempest-cms#10]
I need more input on that, but the main idea is to have a place to showoff people's characters, right?

**Comment by berserkingyadis** [comment]
Yes! For starters a list of the equipment with rarity colors as a permanent link to share would be great. Rendering characters in armor or anything is a bit too crazy for now.
